daemon/container.go

```diff
@@ -330,5 +330,9 @@ func (daemon *Daemon) verifyContainerSettings(platform string, hostConfig *conta
 	}
 
 	// Now do platform-specific verification
-	return verifyPlatformContainerSettings(daemon, hostConfig, config, update)
+	platformWarns, err := verifyPlatformContainerSettings(daemon, hostConfig, config, update)
```

Comment:
Can you check the error here, and return? It "looks" safe in this case, but it's best practice to handle errors;

```go
platformWarns, err := verifyPlatformContainerSettings(daemon, hostConfig, config, update)
if err != nil {
    return nil, err
}
```

Now that `platformWarns` no longer contains "just" platform-specific warnings, we should probably rename it to something more generic (just `warnings`?)